internal/api/configapi/attestationversion.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package configapi
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+const dummyVersionValue = 0
+
+// NewLatestDummyVersion returns the latest version with a dummy version value.
+func NewLatestDummyVersion() AttestationVersion {
+	return AttestationVersion{
+		Value:    dummyVersionValue,
+		IsLatest: true,
+	}
+}
```

Comment:
```suggestion
const placeholderVersionValue = 0

// NewPlaceholderVersion returns a placeholder for the latest version.
func NewPlaceholderVersion() AttestationVersion {
	return AttestationVersion{
		Value:    placeholderVersionValue,
		IsLatest: true,
	}
}
```

To quote the Google developer guide: Don't use dummy to refer to placeholders. Instead, use placeholder.